docs/painless/painless-api-reference/painless-api-reference-aggs/packages.asciidoc

```diff
@@ -0,0 +1,118 @@
+
+[role="exclude",id="painless-api-reference-aggs-org-elasticsearch-xpack-sql-expression-function-scalar-whitelist"]
+=== Aggs API for package org.elasticsearch.xpack.sql.expression.function.scalar.whitelist
+See the <<painless-api-reference-aggs, Aggs API>> for a high-level overview of all packages.
+
+[[painless-api-reference-aggs-InternalSqlScriptUtils]]
```

Comment:
I think we should cut this out of the output. It isn't really "public".

Reply:
I agree with this sentiment.  I don't have a great idea for how to do this and keep it automated.  Any suggestions?

Reply:
For now I've added a hardcoded list that can be added to easily in the meantime.  In the future I would like to make this part of the entire whitelisting process where we can mark a class as internal and have it included in the PainlessLookup.  Possibly, we can even ensure that a user can't execute the method with an internal versus external compilation flag.

Reply:
:+1: